test/storage/offline_database.test.cpp

```diff
@@ -660,41 +671,69 @@ TEST(OfflineDatabase, MigrateFromV3Schema) {
 
     // v3.db is a v3 database, migrated from v2.
 
-    deleteFile("test/fixtures/offline_database/v5.db");
-    writeFile("test/fixtures/offline_database/v5.db", util::read_file("test/fixtures/offline_database/v3.db"));
+    deleteFile("test/fixtures/offline_database/migrated.db");
+    writeFile("test/fixtures/offline_database/migrated.db", util::read_file("test/fixtures/offline_database/v3.db"));
 
     {
-        OfflineDatabase db("test/fixtures/offline_database/v5.db", 0);
+        OfflineDatabase db("test/fixtures/offline_database/migrated.db", 0);
         auto regions = db.listRegions();
         for (auto& region : regions) {
             db.deleteRegion(std::move(region));
         }
     }
 
-    EXPECT_EQ(5, databaseUserVersion("test/fixtures/offline_database/v5.db"));
+    EXPECT_EQ(6, databaseUserVersion("test/fixtures/offline_database/migrated.db"));
 }
 
 TEST(OfflineDatabase, MigrateFromV4Schema) {
     using namespace mbgl;
 
     // v4.db is a v4 database, migrated from v2 & v3. This database used `journal_mode = WAL` and `synchronous = NORMAL`.
 
-    deleteFile("test/fixtures/offline_database/v5.db");
-    writeFile("test/fixtures/offline_database/v5.db", util::read_file("test/fixtures/offline_database/v4.db"));
+    deleteFile("test/fixtures/offline_database/migrated.db");
+    writeFile("test/fixtures/offline_database/migrated.db", util::read_file("test/fixtures/offline_database/v4.db"));
 
     {
-        OfflineDatabase db("test/fixtures/offline_database/v5.db", 0);
+        OfflineDatabase db("test/fixtures/offline_database/migrated.db", 0);
         auto regions = db.listRegions();
         for (auto& region : regions) {
             db.deleteRegion(std::move(region));
         }
     }
 
-    EXPECT_EQ(5, databaseUserVersion("test/fixtures/offline_database/v5.db"));
+    EXPECT_EQ(6, databaseUserVersion("test/fixtures/offline_database/migrated.db"));
 
     // Journal mode should be DELETE after migration to v5.
-    EXPECT_EQ("delete", databaseJournalMode("test/fixtures/offline_database/v5.db"));
+    EXPECT_EQ("delete", databaseJournalMode("test/fixtures/offline_database/migrated.db"));
 
     // Synchronous setting should be FULL (2) after migration to v5.
-    EXPECT_EQ(2, databaseSyncMode("test/fixtures/offline_database/v5.db"));
+    EXPECT_EQ(2, databaseSyncMode("test/fixtures/offline_database/migrated.db"));
+}
+
+
+TEST(OfflineDatabase, MigrateFromV5Schema) {
+    using namespace mbgl;
+
+    // v5.db is a v5 database, migrated from v2, v3 & v4.
+
+    deleteFile("test/fixtures/offline_database/migrated.db");
+    writeFile("test/fixtures/offline_database/migrated.db", util::read_file("test/fixtures/offline_database/v5.db"));
```

Comment:
`v5.db` should be checked in. It's currently ignored in `.gitignore`. I like the rename of the output to `migrated.db` -- that means we can just `migrated.db` and not `v*.db`.